lib/tests/streamlit/elements/arrow_altair_test.py

```diff
@@ -139,7 +139,7 @@ class ArrowChartsTest(DeltaGeneratorTestCase):
     """Test Arrow charts."""
 
     @parameterized.expand(ST_CHART_ARGS)
-    def test_empty_arrow_chart(self, chart_command: Callable, altair_type: str):
+    def test_emptychart(self, chart_command: Callable, altair_type: str):
```

Comment:
nit: I think a bunch of these test names (which I'm guessing were modified via search and replace) had an extra leading or trailing underscore removed around `arrow`.

Same comment applies throughout these unit tests

Reply:
Whoops, yep I just did search & replace. Fixed it 👍